packages/docusaurus-theme-classic/src/theme/BlogTagsListPage/index.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import BlogLayout from '@theme/BlogLayout';
 import TagsListByLetter from '@theme/TagsListByLetter';
 import type {Props} from '@theme/BlogTagsListPage';
-import SearchMetadata from '../SearchMetadata';
```

Comment:
now catched with error

> /Users/sebastienlorber/Desktop/projects/docusaurus/packages/docusaurus-theme-classic/src/theme/BlogTagsListPage/index.tsx
  19:1  error  '../SearchMetadata' import is restricted from being used by a pattern  no-restricted-imports